scripts/generateProptypes.ts

```diff
@@ -204,7 +204,7 @@ async function run(argv: HandlerArgv) {
         folderName = folderName.slice(9);
       }
 
-      return fileName === folderName;
+      return !fileName.endsWith('.test');
```

Comment:
Generated proptypes for all non-test files inside the folder of each component.

Reply:
This is also generating proptypes for components marked as `@ignore` in the JSDoc comments - something to fix?

Reply:
I guess we can try to exclude them if it's easy to do.

Reply:
Also I just noticed every component is not listed in the left sidebar... I'll find a better way.